### PR TITLE
10 final Rooosss

### PR DESCRIPTION
**Pull request by Rooosss**
Link to deployed project - https://rooosss.github.io/fe-course-basic-2023/

### CODE REVIEW

File: docs/rooosss/index.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+
+        <link rel="stylesheet" href="style/style.css">
+        <link rel="stylesheet" href="style/header.css">
+        <link rel="stylesheet" href="style/footer.css">
+        <title>PokemonProject</title>
+    </head>
+    <body>
+        <div class="wrapper">
+            <header class="header">
```
Comment (vladyslav-yermolin-moc) [inline docs/rooosss/index.html:19]: Are these white gaps ok for you?
I suggest moving the background-color to the `.header` and `.footer` correspondently.
Screenshot at Dec 20 22-58-46

File: docs/rooosss/scripts/pokemons.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    });
+    container.appendChild(fragment);
+}
+
+function filterPokemons(pokemonsList) {
+    const isSmallSizeFilterChecked = formFilters.elements.small.checked;
+    const isBigSizeFilterChecked = formFilters.elements.big.checked;
+    const filter = [];
+    pokemonsList.forEach((card) => {
+        if (isBigSizeFilterChecked && card.height >= 100) {
```
Comment (vladyslav-yermolin-moc) [inline docs/rooosss/scripts/pokemons.js:41]: I suggest using some other values, lest say 20'' and 50``. That would add a visual representation of clicking `big` checkbox.
And you can add it as a description to checkboxes texts: `Big(>50'')`

Comment (vladyslav-yermolin-moc) [reply docs/rooosss/scripts/pokemons.js:41]: you can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter for the filtering propose
Also 
> При обраних чекбоксах Big та Small  - покемони нижче 50' та вище 100'

Now when both checkboxes are checked a user can see only small pokemons

Comment (vladyslav-yermolin-moc) [reply docs/rooosss/scripts/pokemons.js:41]: And you can declare the filters values (20 and 50) as constants on the top of the script with some decent name. [Magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) - is an antipattern

File: docs/rooosss/style/footer.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.footer__image-container {
+  margin-bottom: 32px;
+  display: flex;
+  justify-content: center;
+  gap: 30px;
+}
+
+footer__image-container a {
```
Comment (vladyslav-yermolin-moc) [inline docs/rooosss/style/footer.css:16]: this selector is broken and useless. The `a` elements have `cursor: pointer;` by default
image

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]
Good work! Thanks

File: docs/rooosss/pokemons.html [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+                                    <a class="navigation__link" href="login.html">Login</a>
+                                </li>
+                            </ul>
+                        </nav>
+                    </section>
+                </div>
+            </header>
+            <main class="main">
+                <div class="container">
+                    <section class="filters">
```
Comment (vladyslav-yermolin-moc) [inline docs/rooosss/pokemons.html:50]: Same gaps now you have for filters. Proly you can lift up filters block out of main 
image

**Review by vladyslav-yermolin-moc — APPROVED** [review]

**Review by vladyslav-yermolin-moc — APPROVED** [review]

**Comment by vadym-zinchenko-moc** [comment]
Screenshot 2023-12-29 at 18 42 51

I don't see any responsive layout at all

**Comment by vadym-zinchenko-moc** [comment]
Thanks! Really fine project!

**Comment by vadym-zinchenko-moc** [comment]
need to resolve conflict and I can merge it!

**Review by vadym-zinchenko-moc — APPROVED** [review]

**Pull request merged**